ratpack-kafka-producer/src/test/groovy/smartthings/ratpack/kafka/KafkaProducerServiceSpec.groovy

```diff
@@ -381,7 +381,7 @@ class KafkaProducerServiceSpec extends Specification {
 		assert result.throwable.getMessage().contains('Invalid value -1')
 
 		where:
-		property << ['lingersMs', 'batchSize', 'sendBufferBytes', 'maxInFlightRequestsPerConnection', 'bufferMemory']
```

Comment:
From: https://kafka.apache.org/22/documentation.html

send.buffer.bytes | The size of the TCP send buffer (SO_SNDBUF) to use when sending data. If the value is -1, the OS default will be used. | int | 131072 | [-1,...] | medium
| --- | --- | --- | --- | --- | --- |

Reply:
Older versions did not allow negative here but newer versions do.